Automattic/jetpack
# My Jetpack Add filter in my jetpack activation

#### Changes proposed in this Pull Request:

* Added a filter to allow stand-alone plugins to add product specific activation routines

#### Jetpack product discussion


p9dueE-55u-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Test the activation of different products with different combination of plugins installed/active. All should work the same way as before

Testing the hook.

In the Search plugin, add a callback:

```
add_filter( 'my_jetpack_search_activation', function( $result ) {
   if ( is_wp_error( $result ) return $result;
  // do things
  return true;
} );
```
And verify your coded runs when you activate the product via My Jetpack
